torch/_inductor/lowering.py

```diff
@@ -1194,6 +1194,8 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten._fused_moving_avg_obs_fq_helper_functional)
 make_fallback(aten.grid_sampler_2d_backward, require_dense)
 make_fallback(aten.randperm)
+make_fallback(aten._scaled_dot_product_efficient_attention.default)
```

Comment:
could you also add the flash_attention variants as well?
After this PR: https://github.com/pytorch/pytorch/pull/92917 they will start to show up